packages/federation/src/supergraph.ts

```diff
@@ -804,12 +826,75 @@ export function getSubschemasFromSupergraphSdl({
         return res;
       };
     }
+    function visitToReplaceTypeNames(data: any, nameMap: Map<string, string>): any {
```

Comment:
`unknown > any`.  If you're passing something that is really unknown, or can by anything - prefer using `unknown` and then narrowing down the type through typ assertions.
```suggestion
    function visitToReplaceTypeNames(data: unknown, nameMap: Map<string, string>): unknown {
```